crates/driver/src/print_build.rs

```diff
@@ -435,7 +439,14 @@ where
                         };
                         t.entries.push(filegroup_target);
                     } else {
-                        parent_include_src.push(format!("{}", node_file_name));
+                        if is_empty_globs
```

Comment:
this is also duplicated, I wonder if this expression could be a function on something instead.

Reply:
Split up into a function in 9a9ed9cecef933e111d92232c11b257c095e1cc5